ingredient/ingredient-kubernetes/src/plugin.ts

```diff
@@ -0,0 +1,98 @@
+import fs from 'fs'
+import { BaseIngredient, BakeVariable } from "@azbake/core"
+import { promisify } from 'util';
+import { exec as exec_from_child_process } from 'child_process';
+import Replace from 'replace-in-file';
+const exec = promisify(exec_from_child_process);
+
+export class KubernetesPlugin extends BaseIngredient {
+
+    public async Execute(): Promise<void> {
+        const kubeconfigFilename = 'kubeconfig.yaml'

```

Comment:
Should create a random name I think, to support parallel k8s ingredient deployments that might use different configs.